src/libraries/System.IO.FileSystem/tests/ManualTests/ManualTests.cs

```diff
@@ -0,0 +1,90 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.IO;
+using Microsoft.DotNet.XUnitExtensions;
+using Xunit;
+
+namespace System.IO.ManualTests
+{
+    public class FileSystemManualTests
+    {
+        public static bool ManualTestsEnabled => !string.IsNullOrEmpty(Environment.GetEnvironmentVariable("MANUAL_TESTS"));
+
+        [Fact]//[ConditionalFact(nameof(ManualTestsEnabled))]
```

Comment:
Left over from testing?

Reply:
Yes, fixed.
The reason why I was using `[Fact]` is because I noticed that unless the environment variable is set system wide on boot, it won't be respected by the unit tests if I create it in the current terminal instance. I may have to investigate what is causing this in case it's a bug but I will do it separately, since it's unrelated to this change. I still want to leave this public static property as is, since it's how other manual tests were written.